.ci/bump-elastic-stack-snapshot.yml

```diff
@@ -40,5 +40,5 @@ targets:
     scmid: default
     spec:
       file: dev-tools/integration/.env
-      matchpattern: '(ELASTICSEARCH_VERSION=)\d+.\d+.\d+-.*-SNAPSHOT'
-      replacepattern: '$1{{ source "latestVersion" }}-SNAPSHOT'
```

Comment:
for some reason it produced https://github.com/elastic/fleet-server/pull/2338/files#diff-22766f0cd0d5cfcb04aebb38f81f912b6e52b1bc417abc1671954b3c4e8364b2R1